src/css/tabs/auxiliary.css

```diff
@@ -127,15 +145,20 @@
     text-align: center;
 }
 
-.tab-auxiliary .mode .range .channelInfo .limits {
-    padding: 10px 0px;
-}
+/*.tab-auxiliary .mode .range .channelInfo .limits {
```

Comment:
If this is not used any more please remove it.

Reply:
Being unfamiliar with css, I wasn't sure. It was an artifact of testing things out and didn't seem to have any affect on any thing. I will remove.